src/main/java/core/basesyntax/storage/impl/StorageEntry.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.storage.impl;
+
+public class StorageEntry {
+    private String product;
+    private int quantity;
+
+    public StorageEntry(String product, int quantity) {
+        this.product = product;
+        this.quantity = quantity;
+    }
+
+    public String getProduct() {
+        return product;
+    }
+
+    public void setProduct(String product) {
+        this.product = product;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+}
```

Comment:
Consider hiding the inner class elements using access modifiers to follow the encapsulation principle. For example, the fields `product` and `quantity` can be made private and final if they are not intended to be changed after construction, which will make the class immutable and thread-safe.